crypto/keys/src/ed25519.rs

```diff
@@ -236,6 +236,15 @@ impl Verifier<Ed25519Signature> for Ed25519Public {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct Ed25519Private(SecretKey);
 
+impl Clone for Ed25519Private {
+    fn clone(&self) -> Self {
+        Self(
+            SecretKey::from_bytes(self.0.as_ref())
+                .expect("SecretKey to SecretKey should always work"),
+        )
+    }
+}
+
```

Comment:
Clap now requires all arg types to implement clone.
The `mc-consensus-mint-client` uses the `Ed25519Private` directly in its args.
If we feel that allowing one to clone `Ed25519PrivateKey` is a expanding the attack surface then we will probably want to update the `mc-consensus-mint-client` to use `[u8; 32]` in the CLI structs and then convert to `Ed25519PrivateKey` when used. FYI @eranrund

Reply:
Can we perhaps wrap it in a newtype struct if that is the only place that clones it?
I don't think we want a blanket implementation of this

Reply:
that's nicer than what I was thinking then derive From(), I'll work on it, thanks for the suggestion!

Reply:
fwiw I think it's okay to clone private keys

Reply:
Created a newtype struct that clones locally in the mint-client